test/suite/Annotations.test.php

```diff
@@ -580,17 +587,75 @@ protected function testGetAnnotationsFromAnInterface()
 
     protected function testGetAnnotationsFromTrait()
     {
-        if (version_compare(PHP_VERSION, '5.4.0', '<')) {
```

Comment:
Better to check for PHP version in test suite and not depend on inner logic of tested library.